apps/prairielearn/src/tests/institutionAdmins.test.ts

```diff
@@ -0,0 +1,145 @@
+import { assert } from 'chai';
+import { step } from 'mocha-steps';
+import fetch from 'node-fetch';
+import { queryAsync, queryRow } from '@prairielearn/postgres';
+
+import { config } from '../lib/config';
+import helperServer = require('./helperServer');
+import { UserSchema } from '../lib/db-types';
+import { selectUserByUid } from '../lib/user';
+
+const SITE_URL = `http://localhost:${config.serverPort}`;
+const COURSE_URL = `${SITE_URL}/pl/course/1/course_admin/instances`;
+const COURSE_INSTANCE_URL = `${SITE_URL}/pl/course_instance/1/instructor/instance_admin/assessments`;
+const ASSESSMENT_INSTANCES_URL = `${SITE_URL}/pl/course_instance/1/instructor/assessment/1/instances`;
+
+interface AuthUser {
+  name: string;
+  uid: string;
+  uin?: string;
+  isAdministrator?: boolean;
+}
+
+const ADMIN_USER = {
+  name: 'Admin',
+  uid: 'admin@example.com',
+  isAdministrator: true,
+};
+
+const INSTITUTION_ADMIN_USER = {
+  name: 'Institution Admin',
+  uid: 'institution-admin@example.com',
+};
+
+const INSTRUCTOR_USER = {
+  name: 'Instructor',
+  uid: 'instructor@example.com',
+};
+
+async function insertUser(user: AuthUser) {
+  const newUser = await queryRow(
+    `INSERT INTO users (name, uid) VALUES ($name, $uid) RETURNING *;`,
+    {
+      name: user.name,
+      uid: user.uid,
+    },
+    UserSchema,
+  );
+
+  if (user.isAdministrator) {
+    await queryAsync(`INSERT INTO administrators (user_id) VALUES ($user_id);`, {
+      user_id: newUser.user_id,
+    });
+  }
+}
+
+async function withUser<T>(user: AuthUser, fn: () => Promise<T>): Promise<T> {
```

Comment:
I copied this from `enroll.test.ts`. Should probably dedupe this into a new file with testing utilities?

Reply:
This was done long ago in another PR!